styles/style.css

```diff
@@ -158,17 +191,28 @@ p {
   background: var(--orange-dark);
 }
 
-
 /* Article */
 
+.learn-more-articles {
+  display: flex;
+  gap: 1rem;
+  flex-direction: row;
+  flex-wrap: wrap;
+}
+
 .article {
+  display: flex;
+  flex-direction: row;
   margin-bottom: 1rem;
+  border-color: var(--grey-light);
+  border-style: solid;
 }
 
 .article__title {
   margin-bottom: 0.5rem;
   font-size: 1rem;
-  font-weight: 700;
+  font-weight: 600;
+  padding-left: 20px;
```

Comment:
If you have a look at the sample image, do you see how their lines align at the sides? You dont really want your title to stick out the way yours does! 
Their image: 
![image](https://github.com/CodeYourFuture/bikes-for-refugees/assets/118762745/2d17e543-f658-4ca1-afcb-9c6d41415a72)

Yours: 
![image](https://github.com/CodeYourFuture/bikes-for-refugees/assets/118762745/0d8875c3-5cc7-41fc-b994-f3247e2e1ee2)


How can you fix this? Have a look at your padding-left property here!